tests/golden/defaults/appuio-cloud/appuio-cloud/02_validate_namespace_metadata.yaml

```diff
@@ -66,8 +66,10 @@ spec:
             list: 'request.object&& merge(    not_null(request.object.metadata.labels,
               `{}`)   ,not_null(request.oldObject.metadata.labels, `{}`))  | map(&{key:
               @}, keys(@))'
-        message: 'The following labels are allowed: appuio.io/organization, custom.appuio.io/*,
-          kubernetes.io/metadata.name, test.appuio.io/*, compute.test.appuio.io/cpu'
+        message: 'The following labels can be modified: appuio.io/organization, custom.appuio.io/*,
+          kubernetes.io/metadata.name, test.appuio.io/*, compute.test.appuio.io/cpu.
+          Labels given: {{request.object.metadata.labels}}. Labels before modification:
+          {{request.oldObject.metadata.labels}}.'
```

Comment:
How does this look when rendered? A few newlines might improve readability, especially when there are a lot of labels.

Reply:
It's rendered as json. I improved the formatting a bit by including some newlines and some indentation.